packages/angular_devkit/build_angular/src/utils/index-file/inline-fonts.ts

```diff
@@ -0,0 +1,170 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import * as cacache from 'cacache';
+import { readFile } from 'fs';
+import * as https from 'https';
+import { promisify } from 'util';
+import { findCachePath } from '../cache-path';
+import { cachingDisabled } from '../environment-options';
+import { htmlRewritingStream } from './html-rewriting-stream';
+
+const cacheFontsPath = cachingDisabled ? undefined : findCachePath('angular-build-fonts');
```

Comment:
please add type annotation

Reply:
Added, though typically we don't add type annotations in this cases since it's inferred.